config/initializers/o_data.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module OData
+  NAMESPACE = "NEMO"
+  BASE_PATH = "/odata/v1"
+end
```

Comment:
Note: It's in `initializers` because routes.rb depends on NAMESPACE and that seemed to be the only place that routes could safely load from.